lib/Helper/LongRunningHelper.php

```diff
@@ -153,4 +156,19 @@ protected function resolveOptions(string $method, array $options)
 
         return [];
     }
+
+    /**
+     * @internal
+     * Register a temp file which will be deleted on next call of cleanUp()
+     */
+    public static function addTmpFilePath(string $tmpFilePath) {
```

Comment:
I'd prefer to not have this static as well as the `$tmpFilePaths` property, then it's easier to test and consistent with the other stuff.

Reply:
Done